src/gui/src/dbDescriptors.cpp

```diff
@@ -923,4 +956,62 @@ bool DbObstructionDescriptor::lessThan(std::any l, std::any r) const
   return l_obs->getId() < r_obs->getId();
 }
 
+//////////////////////////////////////////////////
+
+std::string DbTechLayerDescriptor::getName(std::any object) const
+{
+  auto layer = std::any_cast<odb::dbTechLayer*>(object);
+  return layer->getConstName();
+}
+
+std::string DbTechLayerDescriptor::getTypeName(std::any object) const
+{
+  return "Tech layer";
+}
+
+bool DbTechLayerDescriptor::getBBox(std::any object, odb::Rect& bbox) const
+{
+  return false;
+}
+
+void DbTechLayerDescriptor::highlight(std::any object,
+                                        Painter& painter,
+                                        void* additional_data) const
+{
+}
+
+Descriptor::Properties DbTechLayerDescriptor::getProperties(std::any object) const
+{
+  auto gui = Gui::get();
+  auto layer = std::any_cast<odb::dbTechLayer*>(object);
+  double dbuPerUU = layer->getTech()->getDbUnitsPerMicron();
+  Properties props({{"Direction", layer->getDirection().getString()},
+                    {"Minimum width", layer->getWidth() / dbuPerUU},
+                    {"Minimum spacing", layer->getSpacing() / dbuPerUU}});
+  const char* micron = "\u03BC";
+  if (layer->getResistance() != 0.0) {
+    props.push_back({"Resistance", convertUnits(layer->getResistance()) + "\u03A9/sq"}); // ohm/sq
+  }
+  if (layer->getCapacitance() != 0.0) {
+    props.push_back({"Capacitance", convertUnits(layer->getCapacitance()*1e-12) + "F/" + micron + "m"});
```

Comment:
layer cap is pF per square micron not micron  (db.h and the conversion in set_dblayer_wire_rc in Resizer.tcl).

clang-format